src/coreclr/jit/lowerloongarch64.cpp

```diff
@@ -152,13 +152,20 @@ GenTree* Lowering::LowerJTrue(GenTreeOp* jtrue)
 
         // LA64's float compare and condition-branch instructions, have
         // condition flags indicating the comparing results.
+        // For LoongArch64, the floating compare result is saved to the specific register,
+        // where there are 8 bits for saveing at most eight different results, that is the FCC0 ~ FCC7.
+        // This is very different with the AArch64 and AMD64.
+        // For AArch64 and AMD64:                         |    // For LoongArch64
+        // cmp  $f1, $f2     <--just compare.             |    fcmp.cond      <--the condition is here.
+        // branch.condition  <--the condition is here.    |    branch true or false by the cc flag.
         if (varTypeIsFloating(cmpOp1))
         {
             op->gtType = TYP_VOID;
             op->gtFlags |= GTF_SET_FLAGS;
             assert(op->OperIs(GT_EQ, GT_NE, GT_LT, GT_LE, GT_GE, GT_GT));
+
             jtrue->SetOper(GT_JCC);
-            jtrue->AsCC()->gtCondition = cond;
+            jtrue->AsCC()->gtCondition = GenCondition::FNE; // For LA64 is only FNE or FEQ.
```

Comment:
It would make more sense to use NE/EQ. These conditions represent the zero flag on other platforms which is much like LA here. The difference between JCC and JCMP is just whether it works on flags or general purpose registers.

Reply:
For LA64, the `GT_JCC` is only used for the floating compare and the branch condition is only EQ or NE.
So I set is FNE or FEQ for indicating the floating type.

Reply:
Ah ok, given that the CPU flags are named fcc0-7 it makes sense to me to represent it with FEQ/FNE instead. 👍

Reply:
I had update it to `EQ` or `NE`, because the branch `bcnez` and `bceqz` are based on the floating comparing result  fcc0~7 which are more liking a intege.

Reply:
@jakobbotsch 
Thanks for your review.